micromanager_gui/_autofocus.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pymmcore_plus import CMMCorePlus, RemoteMMCore
+
+
+class AutofocusDevice:
+    def __init__(self, mmcore: CMMCorePlus | RemoteMMCore = None):
+        super().__init__()
+        self._mmc = mmcore
+
+    @classmethod
+    def set(self, key, mmcore):
```

Comment:
```suggestion
    def create(cls, key, mmcore) -> AutofocusDevice:
```

it's convention to use `cls` instead of `self` in classmethods.  Change name to `create` (we're not really setting anything, we're creating a new object).  then update the usage in main_window

Reply:
(you could also name it `for` if you prefer... as in `AutofocusDevice.for('some_name', mmcore)`